consensus/consensus-types/src/vote.rs

```diff
@@ -62,7 +62,7 @@ impl Vote {
         validator_signer: &ValidatorSigner,
     ) -> Self {
         ledger_info_placeholder.set_consensus_data_hash(vote_data.hash());
-        let signature = validator_signer.sign(&ledger_info_placeholder);
+        let signature = validator_signer.sign(&ledger_info_placeholder).unwrap();
```

Comment:
This will panic, which I don't think we want it to...

Reply:
it doesn't change the existing behavior, this doesn't take input from clients so should be fine

Reply:
It's only used in tests, but just for you I propagated it along